src/lambda/vue/samInvokeEntry.ts

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// temporary, we should be able to generate these entry point files from components
```

Comment:
if the boilerplate is this small, it might be worth the cost to avoid another dependency (if needed)

Reply:
If using multiple components then these 'entry-point' files wouldn't be very boilderplate-y, though all of our webviews only use a single component currently. It'd be nice if the entry-point could be a `.vue` file as well but haven't figure that out yet.